### ISSUE TITLE
Error creating new skill in deployment

### ISSUE THREAD

**Issue by njpollard** [alexa-games/litexa#174]
# Issue Details

This might well be something I've done wrong. I previously had Litexa working. I've now bought a new Mac, installed the latest version of Litexa and can't deploy projects. Thanks in advance for any help.

## Current Litexa Version

0.6.0
aws-cli/2.0.27 Python/3.7.4 Darwin/19.5.0 botocore/2.0.0dev31
ask-cli 2.11.1

## Current Behavior

Output below from deploying an 'empty' project created using **litexa generate**

% Litexa deploy
...
```
[manifest] +1421ms ask-cli version 2.11.1
[manifest] +1ms loading skill.json
[manifest] +2ms building skill manifest
[manifest] +5ms no skillId found in artifacts, creating new skill
[manifest] +2322ms {"message":"[Warn]: This is an asynchronous operation. Check the progress using the following command: ask smapi get-skill-status --skill-id amzn1.ask.skill.aff2ed5e-f0fc-486b-9f89-2d44477717d6 --resource manifest\n"}
[deploy] +5401ms failed manifest deployment
```

It creates a new empty skill which I can see using the ASK developer console, but no code is uploaded.

**Comment by njpollard** [comment]
Update - I've just installed ask-cli@1.7.23 instead, and it seems to work, so presumably this is an incompatibility with ask-cli@2.11.1

**Comment by ayrc** [comment]
Hi njpollard@, 

Sorry we didn't get back to you earlier. The version of Litexa you were using should have had compatibility with ASK CLI v2. We'll keep this open for investigation.

**Comment by amzn-chris-subagio** [comment]
Hey @njpollard thanks for the report. I've been on OSX + litexa 0.6.0 + ask 2.10.2 for a few weeks and haven't seen the same issue. I'll keep looking, and will try with the latest ask. Can you think of anything else that might be a factor?

Thanks!

**Comment by njpollard** [comment]
I can't think of anything, I'm afraid @amzn-chris-subagio. I have replicated the same behaviour on a PC running Windows 10, so I don't think it's related to my setup.

**Comment by amzn-chris-subagio** [comment]
Which version of node are you on?

**Comment by amzn-chris-subagio** [comment]
Actually, never mind, got it now. Should be fixed with #182 Will get it to master and push out a release ASAP.

**Comment by njpollard** [comment]
Thanks for the update @amzn-chris-subagio. Glad it wasn't just me.
